docs/developer/user-permissions.md

```diff
@@ -19,6 +19,18 @@ role or set of permissions that they have. We use a `UserDomainRole`
 `User.domains` many-to-many relationship that works through the
 `UserDomainRole` link table.
 
+## Migrating changes to Analyst Permissions model
+Analysts are allowed a certain set of read/write registrar permissions. 
+Setting user permissions requires a migration to change the UserGroup 
+and Permission models, which requires us to manually make a migration 
+file for user permission changes. 
+To update analyst permissions do the following:
+1. Make desired changes to analyst group permissions in user_group.py.
+2. Follow the steps in 0037_create_groups_v01.py to create a duplicate
```

Comment:
**non blocking suggestion**
```suggestion
2. Follow the steps in the migration file "0037_create_groups_v01.py" to create a duplicate
```

or alternatively, you could make this a link

Reply:
Just added the suggestion - thank you!